### PR TITLE
JSITE-80: Remove old vacancies, add new Middle Python Developer vacancy

### PR DESCRIPTION
**Pull request by stcherenkov**

### CODE REVIEW

File: src/components/ListVacancies/data/data.js [hunk 1]
```diff
@@ -1,104 +1,26 @@
...
-  // {
-  //   link: 'senior-mobile-qa-engineer.html',
-  //   place: 'Saint-Petersburg, Russia',
-  //   position: 'Senior Mobile QA Engineer',
-  //   descr: 'Senior Mobile QA Engineer for an international blockchain startup that develops tools for Banks and Funds to use blockchain & tokens and comply with the SEC and other regulators. Start-up is fully funded for 3 years of development.',
-  // },
   {
-    link: 'senior-ios-developer.html',
+    link: 'financial-controller.html',
+    place: 'Dubai, UAE',
```
Comment (Dragonesis) [inline src/components/ListVacancies/data/data.js]: As I know, we have a hierarchy of positions. New York, St. Petersburg, Dubai

File: src/pages/vacancies/community-manager.html [hunk 2]
```diff
@@ -1,99 +0,0 @@
-<!doctype html>
```
Comment (Dragonesis) [inline src/pages/vacancies/community-manager.html]: If in the near future the update of current vacancies is not planned, then ok, I agree with the removal. And so, there is a folder unpublished-pages, where you can throw vacancies removed from publication

File: src/components/ListVacancies/data/data.js (hunk 1 above)
Comment (stcherenkov) [reply src/components/ListVacancies/data/data.js]: Fixed!

File: src/pages/vacancies/community-manager.html (hunk 2 above)
Comment (stcherenkov) [reply src/pages/vacancies/community-manager.html]: Fixed!

**Review by Dragonesis — APPROVED** [review]

**Pull request merged**